app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -571,11 +574,21 @@ export default {
       500,
       true
     );
+    bus.$on(
+      BUS_EVENTS.NEW_CONVERSATION_MODAL,
+      this.onNewConversationModalActive
+    );
```

Comment:
This implementation is quite leaky, ideally we should be able to handle the drag-drop in the modal without having to disable it here. Let me see if if I get some ideas to do this cleanly

Reply:
@nithindavid I tried reducing it to less than 10, but if we add a z-index of 10, it will cause the same issues. We need to add some z-index in the conversation view screen.

Reply:
An option is to disable drag and drop on the modal, or we can let this go as is and document it is detail in the code along with a TODO comment. @nithindavid what do you think?

Reply:
@scmmishra Resolving this comment as we are going with the current implementation